frontend/src/stores/usePreferencesStore.js

```diff
@@ -6,6 +6,7 @@ const usePreferencesStore = create(persist(set => ({
     theme: 'system',
     color: 'match',
     showGrid: true,
+    ctrlZoom: !navigator.platform?.match(/Win/),
```

Comment:
Might be good to comment this line.

i.e literally this:
<img width="201" alt="image" src="https://user-images.githubusercontent.com/16392483/175763043-52536d27-df7a-4e69-9e06-e21644e415b4.png">